app/controllers/application_controller.rb

```diff
@@ -211,4 +211,22 @@ def logged_in_as(roles)
     end
     has_valid_role
   end
+
+  def sort_feed(wikis)
```

Comment:
I don't think we should put this method in the application controller. Instead we can place it in the `node` model and then call it in the view.